.gitignore

```diff
@@ -1,6 +1,6 @@
-*.aux
-*.log
-*.out
+**.aux
+**.log
+**.out
```

Comment:
Damit die Dateien nicht nur im aktuellen Ordner sondern auch in den Unterordnern ignoriert werden